task/sql-tasks.js

```diff
@@ -130,7 +193,14 @@ async function task_1_8(db) {
  *
  */
 async function task_1_9(db) {
-    throw new Error("Not implemented");
+    let result = await db.query (`
+    SELECT 
+    CustomerID, 
+    ContactName
+    FROM Customers
+    WHERE ContactName LIKE 'F%n%'

```

Comment:
Works as I see. But you can use the `like` operator to meet the condition "**the 4th character is 'n'**"